nix/bootstrap.nix

```
@@ -114,6 +114,7 @@ rec {
         LEAN_CC=${stdenv.cc}/bin/cc ${lean-bin-tools-unwrapped}/bin/leanc -shared ${lib.optionalString stdenv.isLinux "-Bsymbolic"} \
           ${if stdenv.isDarwin then "-Wl,-force_load,${Init.staticLib}/libInit.a -Wl,-force_load,${Lean.staticLib}/libLean.a -Wl,-force_load,${leancpp}/lib/lean/libleancpp.a ${leancpp}/lib/libleanrt_initial-exec.a -lc++"
             else "-Wl,--whole-archive -lInit -lLean -lleancpp ${leancpp}/lib/libleanrt_initial-exec.a -Wl,--no-whole-archive -lstdc++"} -lm ${stdlibLinkFlags} \
+          $(${llvmPackages_14.libllvm.dev}/bin/llvm-config --ldflags) $(${llvmPackages_14.libllvm.dev}/bin/llvm-config --libs) \
```

```suggestion
          $(${llvmPackages_14.libllvm.dev}/bin/llvm-config --ldflags --libs) \
```
works as well I believe

---

Do you still have my Nix commit? I think I avoided the explicit reference to `llvmPackages_14`. Otherwise it looks good.

---

I do not have a reference: I rebased many times, and appear to have lost it in the confusion.

---

I found it